pyspecProcScripts/correlation_alignment.py

```diff
@@ -44,16 +46,16 @@ def correl_align(
         This data is not modified.
     tol:            float
                     Sets the tolerance limit for the alignment procedure.
-    indirect_dim:   str
-                    Name of the indirect dimension along which you seek to
-                    align
-                    the transients.
+    repeat_dims:   list
+                    List of names of the dimensions along which you seek to
+                    align.
```

Comment:
you need to point out that these are dimensions along which signal is essentially **repeated** and therefore can be aligned.  You should also point out that `nScans` is included by default (and do this below), but that a common strategy is to flip the sign .... as I said in the comments.

Reply:
checked